versioned_docs/version-0.1.0/advanced-topics/messaging-system.md

```diff
@@ -33,4 +38,4 @@ A method turned into an RPC is no longer a regular method, it will have its own
 
 ## Network Update Loop
 
-See [Network Update Loop](network-update-loop-system/index.md) for information on the update infrastructure.
\ No newline at end of file
+See [Network Update Loop](network-update-loop-system/index.md) for information on the update infrastructure.
```

Comment:
This was removed in /docs version, should this remain or be removed?

Reply:
removed ill do it now